src/pages/ValidationPage.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable react/jsx-closing-tag-location */
+/* eslint-disable indent */
+import React, { useEffect, useState } from "react";
+import { CardContent, CircularProgress, makeStyles, Button } from "@material-ui/core";
+import { useMobile } from "../utils/media-queries";
+import { Redirect, useParams } from "react-router-dom";
+import { validateApplication } from "../services/companyApplicationService";
+import { getValidationError } from "../components/Apply/Company/CompanyApplicationUtils.js";
+
+const useStyles = (isMobile) => makeStyles((theme) => ({
+    content: {
+        padding: isMobile ? theme.spacing(2, 2) : theme.spacing(3, 3),
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: "center",
+    },
+    title: {
+        color: "rgb(80,80,80)",
+        fontSize: "23px",
+    },
+    text: {
+        color: "rgb(100,100,100)",
+        fontSize: "15px",
+        maxWidth: isMobile ? "100%" : "80%",
+        textAlign: "center",
+
+    },
+    button: {
+        background: "rgb(250,70,70)",
+        color: "white",
+        marginTop: "3vh",
+
+    },
+}));
```

Comment:
This looks like way too much CSS, we will have to analyse this to see what can be replace with Material UI defaults

Reply:
Also, `makeStyles` returns a function that can optionally take arguments, which in this case would be `isMobile` , so there is no need to apply currying here: that argument is available to use if instead of an object you use a function as the style, for instance:

Instead of
```js
text: {
    ...
    maxWidth: isMobile ? "100%" : "80%",
}
```
you could have
```js
text: (isMobile) => ({
    ...
    maxWidth: isMobile ? "100%" : "80%",
})
```

I'm not entirely sure about the syntax but I believe this is a thing. Correct me if I'm wrong.

This is not so much a problem as it is just advice/letting you know.